test/Microsoft.Azure.SignalR.Tests/ConnectionStringParserFacts.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.Azure.SignalR.Tests
 {
-    public class ConnectionStringParsingFacts
+    public class ConnectionStringParserFacts
```

Comment:
extremely nit: why it is *Facts* when it contains *Theories*...

Reply:
I don't think `Facts` in the class name is the same term of `[Facts]` attribute. In the class name, it is basically interchangeable with `Tests`, defining the behavior of `ConnectionStringParser` .